src/features/Founder.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import Avatar from "react-avatar";
+import { useParams } from "react-router-dom";
+import { Flex, Text, Button } from "@chakra-ui/core";
+import db from "../data/database";
+
+function useFounder() {
+  const params = useParams();
+  const [founder, setFounder] = React.useState(null);
+
+  React.useEffect(() => {
+    const user = db.queryAll("users", {
+      query: { ID: params.founder },
+      limit: 1,
+    });
+
+    if (user) setFounder(user);
+  }, [params.founder]);
+
+  return founder ? founder[0] : founder;
+}
+
+function useConnect(username) {
+  const params = useParams();
+  const [isConnected, setConnected] = React.useState(false);
+
+  // check if founders are already connected
+  React.useEffect(() => {
+    const user = db.queryAll("users", {
+      query: { username },
+      limit: 1,
+    });
+
+    if (user?.[0].connections?.includes(params.founder)) {
+      setConnected(true);
+    }
+  }, [params.founder, username]);
+
+  function onConnect() {
+    db.update("users", { username }, (row) => {
+      row.connections = [...(row.connections || []), params.founder];
+      return row;
+    });
+    db.commit();
+    setConnected(true);
+  }
```

Comment:
It just occurred to me that this should update both the current user and the founder